.travis.yml

```diff
@@ -25,6 +26,13 @@ before_install:
 - if $INSTALL_DEB_DEPENDECIES; then source /etc/fsl/fsl.sh; fi
 - if $INSTALL_DEB_DEPENDECIES; then source /etc/afni/afni.sh; fi
 - export FSLOUTPUTTYPE=NIFTI_GZ
+# Install vtk and fix numpy installation problem
```

Comment:
Great, I didn't know. I'll be more than happy to add tests to python 3 and simplify this file. Will update soon, thanks!